clang-tools-extra/test/clang-tidy/checkers/cppcoreguidelines/use-enum-class-ignore-unscoped-enums-in-classes.cpp

```diff
@@ -0,0 +1,18 @@
+// RUN: %check_clang_tidy -std=c++11-or-later %s cppcoreguidelines-use-enum-class %t -- \
```

Comment:
I believe it should be possible to merge these two test files, and provide a specific CHECK-MESSAGES-FULL (or similar) that only warns when the option is not enabled.